.gitpod.yml

```diff
@@ -1,40 +1,54 @@
 github:
   prebuilds:
-    # enable for the master/default branch (defaults to true)
```

Comment:
I generally prefer keeping these comments - though they might seem like visual noise they are helpful to provide context to folks who are not familiar with Gitpod config and might want to make improvements in the future

Reply:
Ah I missed this one.

Sure let's keep them. Would you like to add them back in the follow-up documentation PR?